lib/brcobranca/boleto/template/rghost.rb

```diff
@@ -216,9 +218,14 @@ def modelo_generico_rodape(doc, boleto)
           doc.moveto x: '0.7 cm', y: '10.7 cm'
           doc.show boleto.instrucao6
           doc.moveto x: '1.2 cm', y: '8.8 cm'
-          doc.show "#{boleto.sacado} - #{boleto.sacado_documento.formata_documento}" if boleto.sacado && boleto.sacado_documento
+          doc.show "#{boleto.sacado} - CNPJ/CNPJ: #{boleto.sacado_documento.formata_documento}" if boleto.sacado && boleto.sacado_documento
           doc.moveto x: '1.2 cm', y: '8.4 cm'
           doc.show "#{boleto.sacado_endereco}"
+
+          if boleto.avalista && boleto.avalista_documento
+            doc.moveto x: '2.4 cm', y: '7.47 cm'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.